docs/source/conf.py

```diff
@@ -80,9 +80,9 @@
 # built documents.
 #
 # The short X.Y version.
-version = u'5.0.0'
+version = u'5.0.1'
```

Comment:
Can we pull this from the package setup.py or similar, so it doesn't have to be updated with each release?

Reply:
Good idea. done :+1:

Reply:
Did the commit to do this not get pushed @witekdev?

Reply:
Looks like it did?
https://github.com/Dallinger/Dallinger/pull/1531/commits/58ea1c54a273beca91b51fc82eb08b4b5fba94de

Reply:
^ @alecpm